packages/server/src/sdk/app/rows/search/sqs.ts

```diff
@@ -147,6 +196,8 @@ async function runSqlQuery(
   const response = await alias.queryWithAliasing(json, processSQLQuery)
   if (opts?.countTotalRows) {
     return processRowCountResponse(response)
+  } else if (Array.isArray(response)) {
+    return reverseUserColumnMapping(response)
```

Comment:
Do we need to wrap the `return response` in the else block below in this function?